test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -61,7 +61,7 @@ def run_smdebug_test(
             f"{container_test_local_dir}:{os.path.join(os.sep, 'test')}{shm_setting}{image_uri} "
             f"./{test_script} {framework}",
             hide=True,
-            timeout=2400,
+            timeout=7200,
```

Comment:
this is going to still timeout bc it doesn't account for test setup time. if tf1 cu11 takes a really long time with the existing infra, my suggestion is that we move the test to nightly for tf1 and add faster running functional tests for tf1 in PR context